test/test_dns_unit.cpp

```diff
@@ -68,7 +68,7 @@ TEST_F(DNSTest, TestDecodeDNSstring)
 TEST_F(DNSTest, TestCodeDomain)
 {
   char buffer[16];
-  llarp::Zero(&buffer, 15);
+  llarp::Zero(&buffer, 16);
```

Comment:
should probably be `buffer` not `&buffer`

Reply:
fixed